bucket/scummvm.json

```diff
@@ -9,11 +9,25 @@
     "architecture": {
         "32bit": {
             "url": "https://downloads.scummvm.org/frs/scummvm/2.8.0/scummvm-2.8.0-win32.zip",
-            "hash": "b40771b90567428366153a9ca2a615d92bcf5ce972c0a6633d9e85093974a7df"
+            "hash": "b40771b90567428366153a9ca2a615d92bcf5ce972c0a6633d9e85093974a7df",
+            "pre_install": [
+                "Move-Item -Path \"${dir}\\scummvm-2.8.0-win32\\*\" -Destination \"$dir\" -Force",
+                "Remove-Item \"$dir\\scummvm-2.8.0-win32\"",
```

Comment:
These first two commands can likely be replaced by `extract_dir` which accepts a `$version` placeholder.

Reply:
Ah, ok, that will make things easier, I'll fix it up.

Reply:
Looks like `$version` doesn't get resolved in `extract_dir` here (unlike in `autoupdate`), I have to set the whole name manually. Otherwise, looks like that works :+1: